tests/integration/test_case.go

```diff
@@ -535,6 +535,9 @@ type Request struct {
 	// The request to execute.
 	Request string
 
+	// Name of the operation within the request to execute.
```

Comment:
todo: Please document that this property is optional.

suggestion: I do typically prefer `Option[string]` for optional string properties, I find it is clearer to consumers of the interface.  More than happy for you to shrug this off as personal preference though (please still do the todo though)

Reply:
I prefer string over `Option[string]` here.